src/main/java/org/lanternpowered/server/LanternServerLaunch.java

```diff
@@ -45,7 +45,7 @@
 
 public final class LanternServerLaunch {
 
-    public void main(String[] args) {
+    public static void main(String[] args) {
```

Comment:
You can make this non-static again, the method is called from `LanternLaunch`. This isn't really the `main` method.